composer.json

```diff
@@ -9,8 +9,15 @@
         }
     ],
     "require": {
-        "marco-c/wp_serve_file": "dev-master"
+        "marco-c/wp_serve_file": "dev-master",
+        "stevegrunwell/wp-enforcer": "^0.4.1"
```

Comment:
This package is only needed during development, right? Could we add it to `require-dev` instead of `require`?